apps/comps/components/agents-table/index.tsx

```diff
@@ -515,48 +482,25 @@ export const AgentsTable: React.FC<AgentsTableProps> = ({
       {
         id: "boostPool",
         accessorKey: "boostTotal",
-        header: () => (
-          <span className="whitespace-nowrap">
-            {hasBoostEnabled ? "Boost Pool" : "Votes"}
-          </span>
-        ),
+        header: () => <span className="whitespace-nowrap">Boost Pool</span>,
         cell: ({ row }) => {
-          if (hasBoostEnabled) {
-            const agentBoostTotal = isSuccessAgentBoostTotals
-              ? agentBoostTotals[row.original.id] || 0
-              : 0;
+          const agentBoostTotal = isSuccessAgentBoostTotals
+            ? agentBoostTotals[row.original.id] || 0
+            : 0;
 
-            return (
-              <div className="flex flex-col items-end">
-                <span className="text-secondary-foreground font-semibold">
-                  {isBoostDataLoading
-                    ? "..."
-                    : formatCompactNumber(agentBoostTotal)}
-                </span>
-                <span className="text-xs text-slate-400">
-                  (
-                  {formatPercentage(
-                    Number(agentBoostTotal),
-                    Number(totalBoost),
-                  )}
-                  )
-                </span>
-              </div>
-            );
-          } else {
```

Comment:
how is this going to look in the UI now when viewing old pre-boost comps?

Reply:
The table will have a "Boost pool" column and all agents will have a value of 0. While technically correct, maybe this isn't ideal. If we can find some way of detecting a competition as being pre-boost, then we could hide this column. Thoughts?

Reply:
One option would be to add a column to competitions just to flag them as legacy.